molecule/install-from-package/tests/test_defaults.py

```diff
@@ -0,0 +1,35 @@
+import os
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+def test_libs(host):
+    if host.system_info.distribution == "centos":
+        libs = [
+            "/lib64/libogg.so.0",
+            "/lib64/libmad.so.0",
+            "/lib64/libmp3lame.so.0"
+        ]
+
+    if host.system_info.distribution == "ubuntu":
+        libs = [
+            "/lib/x86_64-linux-gnu/libogg.so.0",
+            "/lib/x86_64-linux-gnu/libmad.so.0",
+            "/lib/x86_64-linux-gnu/libmp3lame.so.0"
+        ]
+
+    for lib in libs:
+        f = host.file(lib)
+        assert f.exists
+        assert f.is_file
```

Comment:
Instead of checking a presense of libraries, I recommend to run `sox --help` command and verify if the requested format is present in `AUDIO FILE FORMATS` list.

When you run sox from console, you will see something like:

```
ubuntu@DESKTOP-X670E:~$ sox --help
sox:      SoX v14.4.2

Usage summary: [gopts] [[fopts] infile]... [fopts] outfile [effect [effopt]]...

AUDIO FILE FORMATS: 8svx aif aifc aiff aiffc al amb amr-nb amr-wb anb au avr awb caf cdda cdr cvs cvsd cvu dat dvms f32 f4 f64 f8 fap flac fssd gsm gsrt hcom htk ima ircam la lpc lpc10 lu mat mat4 mat5 maud mp2 mp3 nist ogg paf prc pvf raw s1 s16 s2 s24 s3 s32 s4 s8 sb sd2 sds sf sl sln smp snd sndfile sndr sndt sou sox sph sw txw u1 u16 u2 u24 u3 u32 u4 u8 ub ul uw vms voc vorbis vox w64 wav wavpcm wv wve xa xi
PLAYLIST FORMATS: m3u pls
AUDIO DEVICE DRIVERS: alsa
```

The following should work (disclaimer: I have not tested it):

```

def test_supported_formats(host):

    help_output = host.check_output(['/usr/bin/sox --help')

    # find a line that starts with "AUDIO FILE FORMATS
    audio_file_formats_line = next((l for l in help_output.lines() if l.startswith("AUDIO_FILE_FORMATS:"), None)
    assert audio_file_formats_line is not None, "AUDIO_FILE_FORMATS is missing in sox --help output"

    # parse the supported formats into a list
    supported_formats = audio_file_formats_line.split(":")[1].split()

    assert "mp3" in supported_formats
    assert "ogg" in supported_formats
    assert "wav" in supported_formats
```

Reply:
This has been implemented